bin/dropletBarcodePlot.R

```diff
@@ -1,33 +1,104 @@
 #!/usr/bin/env Rscript
 
-library(ggplot2)
-library(DropletUtils)
-library(gridExtra)
+suppressPackageStartupMessages(require(optparse))
+suppressPackageStartupMessages(require(ggplot2))
+suppressPackageStartupMessages(require(gridExtra))
+suppressPackageStartupMessages(require(DropletUtils))
+suppressPackageStartupMessages(require(Matrix))
 
-cl <- commandArgs(trailingOnly = TRUE)
+#script obtained from https://github.com/ebi-gene-expression-group/scxa-plots/blob/master/dropletBarcodePlot.R
```

Comment:
To make the droplet barcode plot we need to change the script completely because alevin-fry does not produce the barcode frequencies file that alevin produces. 
So we make the plot from the count matrix instead.